sensor/upgrader/preflight/access.go

```diff
@@ -107,12 +116,76 @@ func (c accessCheck) Check(ctx *upgradectx.UpgradeContext, execPlan *plan.Execut
 		}
 	}
 	if len(actionResourceErr) > 0 {
+		reporter.Errorf("This usually means that access is denied. "+
+			"Have you configured this Secured Cluster for automatically receiving updates? "+
```

Comment:
```suggestion
			"Have you configured this Secured Cluster for automatically receiving upgrades? "+
```
Nit: We're calling it `upgrade` everywhere else, right?

Reply:
good catch!